src/window/window.lisp

```diff
@@ -1,6 +1,7 @@
 (in-package :lem-core)
 
 (define-editor-variable line-wrap t)
+(define-editor-variable impl-source-read-only t)
```

Comment:
This definition is likely to be better located close to the point of use.

Reply:
It's true. I didn't really understand the initialization process of lem, so i put the editor-variable in here.